fractions/opentracing/src/main/java/org/wildfly/swarm/opentracing/runtime/OpenTracingInstaller.java

```diff
@@ -18,12 +21,17 @@
     private final Archive<?> archive;
 
     @Inject
```

Comment:
I think this can be modified to:

`@Inject
private OpenTracingFraction fraction`

as we're inside that fraction so it will always be present for this code